lib/phlex/context.rb

```diff
@@ -76,7 +76,17 @@ def _attributes(attributes)
 
       buffer = first_render ? buffer = +"" : buffer = @_target
 
+      attributes.keys.each do |key|
+        if key.match /[<>&"']/
```

Comment:
Not sure if Ruby interpreter optimizes this out automatically, but if not, would initializing the regex object outside of the loop help to squeeze out a few more CPU cycles?